serde_v8/src/magic/buffer.rs

```diff
@@ -88,7 +96,10 @@ impl serde::Serialize for MagicBuffer {
 
     let mut s = serializer.serialize_struct(BUF_NAME, 1)?;
     let boxed: Box<[u8]> = match self {
-      Self::FromV8(_) => unreachable!(),
+      Self::FromV8(buf) => {
+        let value: &[u8] = &*buf;
```

Comment:
The buffer gets copied here. This seems less optimal - is it intentional?

Reply:
No, this is not intentional; lets see if I can fix this.

Reply:
@inteon I already fixed it, used `&buf` instead